Formula/vcpkg.rb

```diff
@@ -0,0 +1,24 @@
+class Vcpkg < Formula
+  desc "C++ Library Manager for macOS, Linux and Windows"
+  homepage "https://docs.microsoft.com/en-us/cpp/vcpkg"
+  url "https://github.com/Microsoft/vcpkg.git"
```

Comment:
HI and thanks for having a go at creating this Formula. The first issue here is that there are no released versions of this software nor any tags that would allow us to rebuild this formula reliably as is documented in https://docs.brew.sh/Acceptable-Formulae under the section of Stable versions. This means that we can't accept it into homebrew-core but you could of course set up your own tap.